services/Mail.js

```diff
@@ -0,0 +1,45 @@
+import dotenv from 'dotenv';
+import sgMail from '@sendgrid/mail';
+import emailMessages from './mailMessage';
+
+dotenv.config();
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+/**
+ * @exports Mail
+ * @class AuthController
+ * @description Handles the user verification mail sending
+ *
+ */
+class Mail {
+  /**
+   * @static
+   * @param {object} req
+   * @param {object} res
+   * @param {function} next
+   * @return {json} Returns json object
+   * @memberof Mail
+   */
+  static sendVerification(req, res, next) {
+    const { email, token } = req.user;
+    const url = `http://localhost:3000/api/v1/auth/confirmation/${token}`;
```

Comment:
Nice job on your work so far. I think it is better we have this `url` in an environment variable.
You can name it `BASE_URL = localhost:3000/api/v1/` in your .env file and declear it in this file as `const BASE_URL = process.env.BASE_URL`. 
To use it `const url =`http://BASE_URL/auth/confirmation/${token}` `